src/App.tsx

```diff
@@ -1,173 +1,64 @@
-import { Loader } from './components/Loader';
+import { Route, Routes } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Nav } from './components/Nav/Nav';
+import { BlockPeople, ErrorTypes } from './components/BlockPeople/BlockPeople';
+import { Person } from './types';
+import { getPeople } from './api';
 
 import './App.scss';
 
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
+export const App = () => {
+  const [data, setData] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<ErrorTypes | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople().then(fatchData => {
+      setData(fatchData);
+    }).catch(() => {
```

Comment:
not fixed, move it to then block 
```suggestion
    getPeople().then(fatchData => {
      setData(fatchData);
      
       if (data.length === 0) {
        setError(ErrorTypes.NotData);
      }
    }).catch(() => {
```

![image](https://github.com/mate-academy/react_people-table-basics/assets/70009205/d312ed71-8050-490e-bcf8-112fc81a795c)